tests/stacks/ibm/test_data/expected/~g_v1_service_mysql.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
 spec:
   ports:
   - port: 3306
+    protocol: TCP
+    targetPort: 3306
```

Comment:
believe the ports are generic and independent of cloud provider - and accessed internally for mysql/minio

Reply:
@animeshsingh This is the expected output i.e. the results of running `kustomize build`. So if you think this expected output is not "correct" it indicates a bug in the kustomize manifest somewhere.